docs/docs/08.1-more-about-refs.md

```diff
@@ -85,6 +85,22 @@ It's as simple as:
 
   You can access the component's DOM node directly by calling `React.findDOMNode(this.refs.myInput)`.
 
+
+## The ref Callback Attribute
+
+The Ref attribute can be a callback function instead of a name.  This callback will be executed immediately after the component is mounted.  The DOM element associated with the rendered component will be passed in as a parameter, and the callback function may poke the DOM element immediately ("tell it something"), or save the reference for future use (or both).
+
+It's as simple as:
+
+1. Assign a `ref` attribute to anything returned from `render` such as:
```

Comment:
Well, step 2 is to use the return value in the function. Copy & pasting from above doesn't quite work here